### ISSUE TITLE
Docker containers for local testing

### ISSUE THREAD

**Issue by ARMmaster17** [ARMmaster17/Captain#131]
## General Information

- This is a: feature request
- Captain version: `main`
- OS: Debian 10 x64 LXC/Windows 10 x64 + WSL2

## Issue Report

To speed up local testing, and to handle future complex integration tests, it would be nice to have docker containers that could be deployed with Docker Compose (not Helm or K8S manifest since this is not meant to be a permanent service).

## Related issues

Depends on #63
